examples/org.eclipse.emfcloud.modelserver.example/src/main/java/org/eclipse/emfcloud/modelserver/example/CoffeeRouting.java

```diff
@@ -0,0 +1,84 @@
+/********************************************************************************
+ * Copyright (c) 2021 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0, or the MIT License which is
+ * available at https://opensource.org/licenses/MIT.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR MIT
+ ********************************************************************************/
+package org.eclipse.emfcloud.modelserver.example;
+
+import static io.javalin.apibuilder.ApiBuilder.get;
+import static io.javalin.apibuilder.ApiBuilder.path;
+import static org.eclipse.emfcloud.modelserver.emf.common.util.ContextRequest.getIntegerParam;
+import static org.eclipse.emfcloud.modelserver.emf.common.util.ContextRequest.getParam;
+
+import java.net.HttpURLConnection;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.eclipse.emfcloud.modelserver.common.Routing;
+import org.eclipse.emfcloud.modelserver.emf.common.JsonResponse;
+import org.eclipse.emfcloud.modelserver.emf.common.util.ContextResponse;
+import org.eclipse.emfcloud.modelserver.jsonschema.Json;
+
+import com.fasterxml.jackson.databind.node.IntNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.inject.Inject;
+
+import io.javalin.Javalin;
+import io.javalin.http.Context;
+
+public class CoffeeRouting implements Routing {
+   private final Javalin javalin;
+
+   private final AtomicInteger counter = new AtomicInteger(0);
+
+   @Inject
+   public CoffeeRouting(final Javalin javalin) {
+      this.javalin = javalin;
+   }
+
+   @Override
+   public void bindRoutes() {
+      javalin.routes(this::endpoints);
+   }
+
+   protected void endpoints() {
+      path("api/v1", this::coffeeEndpoints);
```

Comment:
```suggestion
      path(ModelServerPathsV1.BASE_PATH, this::coffeeEndpoints);
```

Reply:
Ah, yes, constants, we need to provide AND use them ;-) Thanks!